### PR TITLE
ci: configure actions for fork

### PR DESCRIPTION
**Pull request by cbuto**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -25,9 +24,9 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-      - name: Build container (latest)
-        if: ${{ github.ref == 'refs/heads/master' }}
-        run: 'docker buildx build --push -t ghostunnel/ghostunnel:latest --platform linux/amd64,linux/arm64,linux/arm/v7 .'
+#      - name: Build container (latest)
```
Comment (gracedo) [inline .github/workflows/docker.yml:27]: just wondering what's the reasoning behind commenting this step out (making sure it's intentional)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -40,13 +40,6 @@ jobs:
         uses: actions/checkout@v2
       - name: Run tests 
         run: GO_VERSION=${{ matrix.version }} make docker-test
-      - name: Codecov upload
```
Comment (gracedo) [inline .github/workflows/test.yml:43]: and also just curious why we have to remove these too?

**Review by gracedo — APPROVED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (cbuto) [reply .github/workflows/test.yml:43]: I didn’t want to go through the setup for codecov right now, but maybe we’ll do that in the future (I think you have to setup an account, etc)

File: .github/workflows/docker.yml (hunk 1 above)
Comment (cbuto) [reply .github/workflows/docker.yml:27]: yeah I figured we don’t need a latest image, only the tagged one so I intentionally just commented this out.

Comment (gracedo) [reply .github/workflows/docker.yml:27]: as long as nothing is dependent on that latest image, sgtm!

**Review by palexster — APPROVED** [review]

**Pull request merged**